crates/modelardb_compression/src/models/swing.rs

```diff
@@ -568,23 +700,48 @@ mod tests {
     }
 
     #[test]
-    fn test_can_reconstruct_sequence_of_linear_increasing_values_within_error_bound_zero() {
-        test_can_reconstruct_sequence_of_linear_values_within_error_bound_zero(
+    fn test_can_reconstruct_sequence_of_linear_increasing_values_within_absolute_error_bound() {
+        test_can_reconstruct_sequence_of_linear_values_within_error_bound(
+            ErrorBound::try_new_absolute(ERROR_BOUND_ZERO).unwrap(),
             (42..=4200).step_by(42).map(|value| value as f32).collect(),
         )
     }
 
     #[test]
-    fn test_can_reconstruct_sequence_of_linear_decreasing_values_within_error_bound_zero() {
+    fn test_can_reconstruct_sequence_of_linear_increasing_values_within_relative_error_bound() {
+        test_can_reconstruct_sequence_of_linear_values_within_error_bound(
+            ErrorBound::try_new_relative(ERROR_BOUND_ZERO).unwrap(),
+            (42..=4200).step_by(42).map(|value| value as f32).collect(),
+        )
+    }
+
+    #[test]
+    fn test_can_reconstruct_sequence_of_linear_decreasing_values_within_absolute_error_bound() {
         let mut values: Vec<Value> = (42..=4200).step_by(42).map(|value| value as f32).collect();
         values.reverse();
 
-        test_can_reconstruct_sequence_of_linear_values_within_error_bound_zero(values);
+        test_can_reconstruct_sequence_of_linear_values_within_error_bound(
+            ErrorBound::try_new_absolute(ERROR_BOUND_ZERO).unwrap(),
+            values,
+        );
     }
 
-    fn test_can_reconstruct_sequence_of_linear_values_within_error_bound_zero(values: Vec<Value>) {
+    #[test]
+    fn test_can_reconstruct_sequence_of_linear_decreasing_values_within_relative_error_bound() {
+        let mut values: Vec<Value> = (42..=4200).step_by(42).map(|value| value as f32).collect();
+        values.reverse();
+
+        test_can_reconstruct_sequence_of_linear_values_within_error_bound(
+            ErrorBound::try_new_relative(ERROR_BOUND_ZERO).unwrap(),
+            values,
+        );
+    }
+
+    fn test_can_reconstruct_sequence_of_linear_values_within_error_bound(
```

Comment:
Maybe rename this to something with `assert_` instead of `test_` since `test_` makes it seem like this is an actual unit test.

Reply:
Fixed.